services/venice-controller/src/test/java/com/linkedin/venice/controller/kafka/consumer/TestAdminConsumerService.java

```diff
@@ -103,4 +115,66 @@ public void testMultipleAdminConsumerServiceWithSameMetricsRepo() {
       Utils.closeQuietlyWithErrorLogged(adminConsumerService2);
     }
   }
+
+  @Test
+  public void testCreateKafkaConsumerWithExpectedKafkaProperties() throws Exception {
```

Comment:
unrelated test?

Reply:
I’ve updated the client ID used by the admin consumption service. CC flagged a lack of test coverage for the change, so I added corresponding unit tests.